src/routes/key_value.py

```
@@ -0,0 +1,52 @@
+kv_router = APIRouter(prefix="/kv", tags=["kv"])
+
+
+class KvUpdateBody(BaseModel):
+    value: str
+
+
+@kv_router.get("/{key}")
+async def get_kv_value(
+    key: str,
+    session: SessionDep,
+    request: Request,
```

Ruff (0.13.3)

Unused function argument: `request`

(ARG001)